src/i18n/DopplerIntlProvider.js

```diff
@@ -21,3 +21,5 @@ export default ({ locale, children }) => {
     </IntlProvider>
   );
 };
+
+export default DopplerIntlProvider;
```

Comment:
La idea no es omitir los default exports?

```suggestion
export DopplerIntlProvider;
```

Reply:
No, si son defaults tiene que tener nombres, justo en este caso se complicaba volverlo named.

Reply:
Ok, gracias!